test/functional/wallet_migration.py

```diff
@@ -40,7 +40,6 @@ def set_test_params(self):
         self.setup_clean_chain = True
         self.num_nodes = 2
         self.supports_cli = False
-        self.extra_args = [[], ["-deprecatedrpc=create_bdb"]]
```

Comment:
In fadaa8baafc27fbe29721300569e58777643bb1f "test: remove unused extra_args"

These args are used as the 28.0 node used in this test needs them to make the legacy wallets for testing.

Reply:
Indeed, the second node here is v28 and the `wallet_migration` test fails in the CI with this error:

```python
test_framework.authproxy.JSONRPCException: BDB wallet creation is deprecated and will be removed in a future release. In this release it can be re-enabled temporarily with the -deprecatedrpc=create_bdb setting. (-4)
```